src/main/resources/topics-and-qrels/topics.covid-round1-udel.xml

```diff
@@ -0,0 +1,152 @@
+<topics batch="1" task="COVIDSearch 2020">
+  <topic number="1">
+    <query>coronavirus origin origin COVID-19</query>
+    <question>what is the origin of COVID-19</question>
```

Comment:
What if we just kept `query` and removed `question` and `narrative` - just to reduce confusion?